doc/man3/OPENSSL_CTX.pod

```diff
@@ -32,19 +34,23 @@ OPENSSL_CTX_load_config() loads a configuration file using the given C<ctx>.
 This can be used to associate a libctx with providers that are loaded from
 a configuration.
 
-OPENSSL_CTX_free() frees the given C<ctx>.
+OPENSSL_CTX_free() frees the given I<ctx>.
+
+OPENSSL_CTX_set0_default() sets the default OpenSSL library context to be
+I<libctx> in the current thread.  The previous default library context is
+returned.
```

Comment:
Maybe we should recommend that it is restored to its old value via a subsequent call to OPENSSL_CTX_set0_default() using the previously returned libctx.

Reply:
Done